Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -239,38 +239,25 @@ def parse_github_release_data(release_data: dict) -> K8sRelease:
 
 @dataclass(frozen=True, eq=True)
 class VersionRange:
-    """Version range with a lower and upper bound."""
+    """
+    Version range with a lower and upper bound. Supports checking if a
```

Comment:
```suggestion
    Version range with a lower and upper bound.

    Supports checking if a
```

I think PEP-8 says to use the first line for the summary only.